WordPress/Classes/Models/ReaderDefaultTopic.swift

```diff
@@ -4,4 +4,26 @@ import Foundation
     override open class var TopicType: String {
         return "default"
     }
+
+    /// - note: Starting with 25.5, the app no longer uses the default topics
+    /// fetched from `/read/menu`. Instead, it uses the topics created locally.
+    /// The only reason these topics are needed is to filter entities in Core Data.
+    static func make(path: Path, in context: NSManagedObjectContext) throws -> ReaderDefaultTopic {
+        if let topic = context.firstObject(ofType: ReaderDefaultTopic.self, matching: NSPredicate(format: "path == %@", path.rawValue)) {
+            return topic
+        }
+        let topic = ReaderDefaultTopic(entity: ReaderDefaultTopic.entity(), insertInto: context)
+        topic.type = TopicType
+        topic.path = path.rawValue
+        topic.title = ""
+        try context.save()
+        return topic
+    }
+
+    enum Path: String {
+        // These are arbitrary values just to identify the entites.
+        case recommended = "/jpios/discover/recommended"
+        case firstPosts = "/jpios/discover/first-posts"
+        case latest = "/jpios/discover/latest"
+    }
```

Comment:
Does it mean there should be three unique "default topics" (a.k.a "Discover" topics?) in Core Data? If that's the case, should we delete `-[ReaderTopicService defaultTopicForRemoteTopic:inContext:]` to avoid arbitrary `ReaderDefaultTopic` being created?

Reply:
There are two that are still remaining: one for "Recent" and one for "Likes", so I can't remove them just yet. The "Discover" topic will also be in the database for now, but the app never displays them anywhere, so it's going to be fine. My plan is to ultimately remove `ReaderDefaultTopic` entirely. There are no good reasons to store them in Core Data.

Reply:
Btw, this PR also fixes the following issue:

**Steps**
- Open Discover
- Select "Latest Posts" and wait until they load
- Terminate the app
- Reopen Discover (it shows "Recommended" again)

**Expected result:**
- It loads new posts

**Actual result:**
- It shows posts cached for "Latest Posts", then reloads